package tasks.exception;

public class UserAuthException extends Exception {

    public UserAuthException(String message) {
        super(message);
    }
}
